bls-runtime/src/plog.rs

```diff
@@ -0,0 +1,43 @@
+use std::{fmt::Arguments, sync::Once};
+
+use env_logger::Logger;
+use log::{Level, Log, MetadataBuilder, Record};
+
+static mut ENV_LOGGER: Option<Logger> = None;
```

Comment:
Lets try not use an option type - but instead use the concrete type directly.
We can use `OnceLock` for this.

Example (i'd envision something like this):
```rust
static mut ENV_LOGGER: std::sync::OnceLock<Logger> = std::sync::OnceLock::new();
pub fn env_logger() -> &'static mut Logger {
    ENV_LOGGER.get_or_init(|| { ... })
}
```